src/app/application.cpp

```diff
@@ -907,13 +907,13 @@ int Application::exec()
 #ifndef DISABLE_WEBUI
         m_webui = new WebUI(this);
 #ifdef DISABLE_GUI
-        if (m_webui->isErrored())
-            QCoreApplication::exit(EXIT_FAILURE);
-        connect(m_webui, &WebUI::fatalError, this, []() { QCoreApplication::exit(EXIT_FAILURE); });
-
         printf("%s", qUtf8Printable(u"\n******** %1 ********\n"_s.arg(tr("Information"))));
 
-        if (m_webui->isEnabled())
+        if (m_webui->isErrored())
+        {
+            printf("%s\n", qUtf8Printable(tr("Failed to start the WebUI! To fix the error, you may need to edit the config file manually.")));
```

Comment:
```suggestion
            fprintf(stderr, "%s\n", qUtf8Printable(tr("Failed to start the WebUI! To fix the error, you may need to edit the config file manually.")));
```

Reply:
I tried this initially, but it simply does not compile.